tensorboard/webapp/plugins/npmi/views/main/main_component.scss

```diff
@@ -0,0 +1,46 @@
+/* Copyright 2020 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+@import 'tensorboard/webapp/theme/tb_theme';
+
+:host {
+  display: flex;
+  flex-direction: column;
+  height: 100%;
+}
+
+.content {
```

Comment:
do you need `.content`? Can you just omit the div and add these rules to the `:host`?

Reply:
I will need it later. My current Main Component looks more like this:
```
<div class="zoom-view" *ngIf="annotationsZoomed; else noZoom">
  <npmi-annotations></npmi-annotations>
</div>
<ng-template #noZoom>
  <npmi-metric-arithmetic></npmi-metric-arithmetic>
  <div class="content">
    <div class="selection-container">
      <div class="run-selector">
        <tb-legacy-runs-selector></tb-legacy-runs-selector>
      </div>
      <npmi-metrics-overview class='metrics-filters'></npmi-metrics-overview>
    </div>
    <div *ngIf="runActive === true; else noRun" class="analysis-container">
      <npmi-pc-detail class="pc-detail"></npmi-pc-detail>
      <npmi-selected-annotations class="selected-annotations"></npmi-selected-annotations>
    </div>
    <ng-template #noRun>
      You need to select at least one run.
    </ng-template>
  </div>
</ng-template>
```
Might change, but I think there I will need the container. Should I remove it for now and add it back later?